README.md

```diff
@@ -167,6 +167,15 @@ This container doubles as a skeleton for creating proxy configuration around
 Zipkin like authentication, dealing with CORS with zipkin-js apps, or
 terminating SSL. 
 
+If you want to run the zipkin-ui standalone against a remote zipkin server, you
+need to set `ZIPKIN_BASE_URL` accordingly:
+
+```bash
+$ docker run -d -p 80:80 \
+  -e ZIPKIN_BASE_URL=http://myfavoritezipkin:9411 \
```

Comment:
note: this variable is the same name as what's in the javascript, but it is evaluated only in the context of nginx. As far as the browser is concerned, zipkin is on the same origin.